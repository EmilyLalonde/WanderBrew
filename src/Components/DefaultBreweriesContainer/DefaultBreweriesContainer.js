import React from 'react'
import Card from '../Card/Card'

const DefaultBreweriesContainer = ({denverBreweries}) => {
  const denBreweries = denverBreweries.map(brewery => {
    return <Card 
    name={brewery.name}
      street={brewery.street} 
      city={brewery.city}
      state={brewery.state}
      phone={brewery.phone}
      key={brewery.id}
      id={brewery.id}
    />
  })
  return (
    <div>
      {denBreweries}
    </div>
  )
}

export default DefaultBreweriesContainer