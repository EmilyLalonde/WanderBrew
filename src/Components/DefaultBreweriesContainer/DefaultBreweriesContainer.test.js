import React from 'react'
import { shallow } from 'enzyme'
import  DefaultBreweriesContainer  from './DefaultBreweriesContainer'

describe('DefaultBreweriesContainer', () => {
  let wrapper;
  let mockBreweries;

  beforeEach(() => {
    mockBreweries = [
      {
      id: 1516,
      name: 'Ratio Beerworks',
      street: '2920 Larimer St',
      city: 'Denver',
      state:'Colorado',
      phone:'3012669264',
      }
    ]

    wrapper = shallow(<DefaultBreweriesContainer breweries={mockBreweries}/>)
  });

  it('should match the snapshot with all data being passed correctly', () => {
    expect(wrapper).toMatchSnapshot();
  })
  
})