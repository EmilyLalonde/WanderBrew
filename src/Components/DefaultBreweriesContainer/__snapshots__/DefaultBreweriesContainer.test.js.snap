// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DefaultBreweriesContainer should match the snapshot with all data being passed correctly 1`] = `
<main
  className="default-page"
>
  <Link
    key="1516"
    to="/home/1516"
  >
    <Card
      city="Denver"
      id={1516}
      key="1516"
      name="Ratio Beerworks"
      phone="3012669264"
      state="Colorado"
      street="2920 Larimer St"
    />
  </Link>
</main>
`;
